### ISSUE TITLE
History entry selection can't be expanded using Shift+Arrow

### ISSUE THREAD

**Issue by AlexanderSher** [Microsoft/RTVS#3271]
- If entry is selected, Shift+Up/Down should change selected entries
- If text is selected, Shift+Up/Down should change selection similar to the editor
- If several entries are selected, Shift+Up/Down should be similar to Shift+Mouse (see also https://github.com/Microsoft/RTVS/issues/3256)
